FrontEnd/FrontEnd.Web/Components/Pages/Users/SingleUser.razor

```diff
@@ -24,7 +26,15 @@ else
         <FluentPersona Image="@_user.ImageUrl" ImageSize="200px" Style="margin: 0 auto;"/>
         <FluentLabel Typo="Typography.H1" MarginBlock="10px" Alignment="HorizontalAlignment.Center" Style="width: 100%">@_user.Username</FluentLabel>
         <FluentLabel Typo="Typography.H3">Recipes</FluentLabel>
-        <FluentLabel Typo="Typography.Body">Recipes should be shown here</FluentLabel>
+        @if (_userRecipes != null)
```

Comment:
Could we handle empty _userRecipes case here?

Reply:
I'll take a shortcut :D will not show recipes header at all